pegasus/sites.v3/code.org/public/sports.haml

```diff
@@ -1,15 +1,15 @@
 ---
-title: Sports Hour of Code
+title: "<%= I18n.t(:sports_page_title) %>"
 theme: responsive
 social:
   "twitter:card": photo
   "twitter:site": "@codeorg"
   "twitter:url": "https://code.org/sports"
   "twitter:image:src" : "https://code.org/images/sports/basketball.png"
-  "twitter:title": "Sports Hour of Code Tutorials"
```

Comment:
My mistake on these. Never got the social tags officially approved (just the screenshot of the page was approved) so I'd double check with Jon.